config/private_key.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2022 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package config
+
+import (
+	"net/http"
+	"os"
+	"time"
+
+	vault "github.com/hashicorp/vault/api"
+	"github.com/iotexproject/iotex-core/blockchain"
```

Comment:
move `/iotex-core/xxx` to after L18 as a 3rd block, you can refer to other files